geoalchemy2/admin/dialects/geopackage.py

```diff
@@ -64,6 +71,17 @@ def load_spatialite_gpkg(dbapi_conn, connection_record):
     dbapi_conn.execute("SELECT EnableGpkgAmphibiousMode();")
```

Comment:
I think this line (`EnableGpkgAmphibiousMode`) should go to `load_geopackage_driver` as it should be called *always* rather than just on initialization of the file.

Reply:
Good point, I moved it.